### PR TITLE
Feature/cax portal cplp 417 document upload

### PR DESCRIPTION
**Pull request by vsamarora**
Stepper Component changes added loop to display steps
User Handling on page change
Custodian Data Submission
document upload API

### CODE REVIEW

File: portal/code/company-portal/src/components/cax-responsibilities.tsx [hunk 1]
```diff
@@ -142,7 +138,9 @@ export const ResponsibilitiesCax = ({
   } else {
     toast.error("Email or User Role empty.");
```
Comment (oyo) [inline portal/code/company-portal/src/components/cax-responsibilities.tsx:139]: i18n missing?

File: portal/code/company-portal/src/components/dragdrop.tsx [hunk 2]
```diff
@@ -18,16 +19,22 @@ interface DragDropProps {
 export const DragDrop = ({currentActiveStep, addCurrentStep}: DragDropProps) => {
   const { t } = useTranslation();
 
+  useEffect(() => {
+    window.scrollTo(0, 0)
+  }, [])
+
     // Payload data and url to upload files
     const getUploadParams = ({ meta }) => { return { url: 'https://httpbin.org/post' } }
```
Comment (oyo) [inline portal/code/company-portal/src/components/dragdrop.tsx]: I assume this is just a placeholder until the real API is ready

File: portal/code/company-portal/src/components/stepper.tsx [hunk 3]
```diff
@@ -17,142 +17,47 @@ import "react-datepicker/dist/react-datepicker.css";
...
 
 export const Stepper: FC<StepperProps> = ({currentActiveStep}) => {
-    const { t } = useTranslation();
-    console.log(currentActiveStep)
     return (
         <div className="mx-auto col-11 reg-steps">
               <Row className="stepper-wrapper row-cols-5">
-                <div className="stepper-item completed col">
+             {
+               Object.entries(stepNames).map(element => {
```
Comment (oyo) [inline portal/code/company-portal/src/components/stepper.tsx:30]: good! :)

**Review by oyo — APPROVED** [review]
minor remarks but generally approved

File: portal/code/company-portal/src/components/cax-responsibilities.tsx (hunk 1 above)
Comment (vsamarora) [reply portal/code/company-portal/src/components/cax-responsibilities.tsx:139]: will handle i18n in this https://jira.catena-x.net/browse/CPLP-597

File: portal/code/company-portal/src/components/dragdrop.tsx (hunk 2 above)
Comment (vsamarora) [reply portal/code/company-portal/src/components/dragdrop.tsx]: fixed changes

**Pull request merged**